File: CPP/Program/Fruit.cpp
#pragma once
#include <iostream>
#include <string>
using namespace std;

class Fruit {
protected:
    string name;
    string color;

public:
    Fruit(const string &n, const string &c) : name(n), color(c) {}

    void setName(const string &n) { name = n; }
    string getName() const { return name; }

    void setColor(const string &c) { color = c; }
    string getColor() const { return color; }

    void display() const {
        cout << "FRUIT\n";
        displayInfo();
    }

    void displayInfo() const {
        cout << "> Fruit Name: " << name << endl;
        cout << "> Fruit Color: " << color << endl;
    }
};

File: Java/Program/Fruit.java

abstract class Fruit {
    protected String name;
    protected String color;

    public Fruit(String name, String color) {
        this.name = name;
        this.color = color;
    }

    public String getName() {
        return name;
    }

    public String getColor() {
        return color;
    }

    public void displayInfo() {
        System.out.println("> Fruit Name: " + name);
        System.out.println("> Fruit Color: " + color);
    }
}